.github/workflows/ci.yml

```diff
@@ -15,7 +15,7 @@ env:
   LC_ALL: en_US.UTF-8
 
 concurrency:
-  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
+  group: ${{ github.workflow }}-${{ github.head_ref || github.sha }}
```

Comment:
Was there a reason for this change?  Generally I'll ask that about small changes like this because unless there's a purpose, minimize the number of changes in the PR.